vrecord

```diff
@@ -1257,6 +1257,16 @@ if [[ ! -s "${VRECORD_OUTPUT}" ]] ; then
         _writeingestlog "FFmpeg command Error" "${P1_ERR}"
         exit 1
     fi
+    if [[ "$P2_ERR" != "0" && ! -z "$P2_ERR" ]] ; then
+        _report -wts "Error: The second component of vrecording recording process gave an Error Code - ${P2_ERR}"
```

Comment:
I’m not English speaker, but `component of vrecording recording process` sound stage to me. Is `component of vrecord recording process` not sufficient?
(also below)